book/website/ethical-research/activism/activism-unionisation.md

```diff
@@ -180,6 +180,26 @@ We recognise that the list below is very UK-centric and would welcome additions
 #### Ireland
 * **Medical Laboratory Scientists Association (MLSA)**, Ireland, is the sole trade union representing over 1600 Medical Scientists in Ireland [website](https://mlsa.ie/who-we-are)
 
+#### France
+* **Confédération Française Démocratique du Travail (CFDT)** is the largest French trade union confederation by number of members. It originated as a laic branch of the Christian trade union Confédération Française des Travailleurs Chrétiens (CFTC) [website](https://cfdt.fr/)
```

Comment:
I think we need to find a different word for laic as I had to look this up - perhaps change this to "secular"?

Reply:
Hi @Arielle-Bennett , that sounds great. I will replace laic with secular :)